NASA-PDS/registry-mgr
# Update product's archive status

## 🗒️ Summary
(1) Update product's archive status - individual products, collections and primary references, bundles.
(2) Fixed "ASCII_Date_Time_DOY_UTC" mapping

## ⚙️ Test Data and/or Report
- Create new registry
- Harvest and load some data (a bundle with collections and products)
- Run registry manager:
`registry-manager set-archive-status -status archived -lidvid "urn:nasa:pds:kaguya_grs_spectra:document::1.0"`
    where lidvid can be a product, collection, or a bundle.
- Check that each Elasticsearch document has the following field and value: 
`"ops:Tracking_Meta/ops:archive_status":"archived"`

NOTE: When updating a status of a collection, all its PRIMARY references from the inventory should be updated.
When updating a status of a bundle, all its collections (and primary references) should be updated.

## ♻️ Related Issues
(1)
https://github.com/NASA-PDS/registry-harvest-cli/issues/2
https://github.com/nasa-pds/registry-harvest-service/issues/12
https://github.com/NASA-PDS/registry-harvest-cli/issues/8
(2)
https://github.com/NASA-PDS/pds-registry-common/issues/20
